hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/IOUtils.java

```diff
@@ -256,7 +255,7 @@ public static void skipFully(InputStream in, long len) throws IOException {
    * instead
    */
   @Deprecated
-  public static void cleanup(Log log, java.io.Closeable... closeables) {
+  public static void cleanup(Logger log, java.io.Closeable... closeables) {
```

Comment:
i'd prefer the method to be completely cut and mvoed to cleanupWithLogger; they are now the same methods so you can just invoke that from this and the use the IDE to remove the method

Reply:
Updated it to use `cleanupWithLogger(log, closeables)` directly. Since the class is IA.Public, maybe we could wait a little longer before removing it completely?

Reply:
but we've just changed the signature, haven't we? any application which calls the existing method is going to fail to link

Reply:
Yes that's valid. I am fine with any of these options:
1. We remove this method with this PR
2. We remove this method with a sub-task
3. We let it be here for a while and that would anyways mandate downstreamers to migrate to slf4j for using this utility

Reply:
* for 3.3.5+ do a pr which tags as deprecated "will be cut" for early warning.
* then do 1 or 2 here.

Reply:
Updated the javadoc, will create sub-task for HADOOP-16206 to get this removed.